configs/Asus/SABERTOOTH-P67.CONF

```diff
@@ -0,0 +1,93 @@
+# lm-sensors configuration for the ASUS Sabertooth P67 motherboard
+#
+# Manufacturer: ASUSTeK COMPUTER INC.
+# Product Name: Sabertooth P67
+# Version: Rev 1.2
+# Super I/O: NCT6776
+# UEFI versions tested: 2302
+#
+# Recommended drivers: coretemp, nct6776
+
+# Piet van Duivenbode, 2021
+#
+
+chip "coretemp-*"
```

Comment:
Not sure if CPU monitors should be here, as this should really only be about the motherboard itself